pages/settings/teams.tsx

```diff
@@ -153,4 +166,16 @@ export default function Teams(props) {
       </SettingsShell>
     </Shell>
   );
-}
\ No newline at end of file
+}
+
+// Export the `session` prop to use sessions with Server Side Rendering
+export const getServerSideProps: GetServerSideProps<{ session: Session | null }> = async (context) => {
+    const session = await getSession(context);
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689300)
```suggestion
  const session = await getSession(context);
```